package.json

```
@@ -31,10 +31,11 @@
   },
   "homepage": "https://github.com/MetaMask/extension-port-stream#readme",
   "dependencies": {
-    "webextension-polyfill-ts": "^0.26.0"
+    "webextension-polyfill": ">=0.10.0"
   },
   "devDependencies": {
     "@metamask/eslint-config": "^4.1.0",
+    "@types/webextension-polyfill": ">=0.10.0",
```

Nit: Could we use a `^` range here instead? Seems more conventional and avoids post-v1 breaking changes in theory.


```suggestion
    "webextension-polyfill": "^0.10.0"
  },
  "devDependencies": {
    "@metamask/eslint-config": "^4.1.0",
    "@types/webextension-polyfill": "^0.10.0",
```

---

This seems OK to me, though I'm not sure why this would be preferable to the more conventional `^0.10.0` range